xdsl/utils/marimo.py

```diff
@@ -36,10 +36,15 @@ def pipeline_html(
     ctx: MLContext, module: ModuleOp, passes: Sequence[tuple[mo.Html, ModulePass]]
 ) -> tuple[MLContext, ModuleOp, mo.Html]:
     """
-    Returns a tuple of the resulting module after applying the passes, and the
+    Returns a tuple of the resulting context and module after applying the passes, and the
     Marimo-optimised representation of the modules throughout compilation.
```

Comment:
what does "marimo-optimized" mean?

Reply:
I meant "prettier to view than `str(bla)`", but I was conscious of it not being a great way to phrase this idea. I couldn't find a term that they used themselves for this concept that isn't just the type of the return value.